daffodil-lib/src/main/resources/org/apache/daffodil/xsd/tdml.xsd

```diff
@@ -151,12 +151,22 @@
     <attribute name="root" type="xs:NCName" use="optional"/> <!-- only needed when there is no infoset. -->
     <attribute name="model" type="xs:string" use="optional"/> <!-- is there a type for a path/uri? -->
     <attribute name="config" type="xs:string" use="optional"/> <!-- is there a type for a path/uri? -->
-    <attribute name="roundTrip" type="xs:boolean" use="optional"/>
+    <attribute name="roundTrip" type="tns:roundTripType" use="optional"/>
     <attribute name="description" type="xs:string" use="optional"/>
     <attribute name="unsupported" type="xs:boolean" use="optional" default="false"/>
     <attribute name="validation" type="tns:validationType" use="optional"/>
   </attributeGroup>
 
+  <simpleType name="roundTripType">
+    <restriction base="xs:token">
+      <enumeration value="false"/> <!-- means same as none -->
+      <enumeration value="none"/>
+      <enumeration value="true"/> <!-- means same as simple -->
+      <enumeration value="simple"/> <!--  parse, compare infoset, unparse - compare data -->
+      <enumeration value="twoPass"/> <!-- parse, unparse, reparse - compare infoset -->
```

Comment:
Thoughts on "onePass" instead of "simple" just for consistency with "twoPass", and it's maybe a bit more clear what it's doing?

Reply:
I agree, onePass is more clear.

Reply:
Agreed. This should be changed. Will make the documetnation of the feature cleaner too.